actionpack/lib/action_dispatch/middleware/stack.rb

```diff
@@ -151,10 +151,9 @@ def move_after(target, source)
       middlewares.insert(target_index + 1, source_middleware)
     end
 
-    def use(klass, *args, &block)
-      middlewares.push(build_middleware(klass, args, block))
+    def use(...)
+      middlewares.push(build_middleware(...))
```

Comment:
This is not equivalent. See the CI build failures.

Reply:
Thanks for pointing that out. I fixed that one.